code/datums/ai/robot_customer/robot_customer_behaviors.dm

```diff
@@ -69,7 +72,7 @@
 		finish_action(controller, FALSE)
 		return
 
-	if(DT_PROB(1, delta_time))
+	if(DT_PROB(0.1, delta_time))
```

Comment:
again, why not have a slightly reasonable time on this rather than literaly dividing it by 10?

Reply:
also remove dt_prob from this while you're at it, that probably skewed the chance.

Reply:
See above. With better delta_time values, now includes the maths for expected percentage chance to have vocalised at least once per minute.